lib/Chunk.js

```diff
@@ -160,7 +160,7 @@ class Chunk {
 	}
 
 	mapModules(fn) {
-		return Array.from(this._modules, fn);
+		return Array.from(this._modules).map(fn);
```

Comment:
This should not make any difference.


from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

> More clearly, Array.from(obj, mapFn, thisArg) has the same result as Array.from(obj).map(mapFn, thisArg), except that it does not create an intermediate array.